src/Tizen.Security.WebAuthn/doc/api/Tizen.Security.WebAuthn.md

```diff
@@ -0,0 +1,25 @@
+---
+uid: Tizen.Security.WebAuthm
```

Comment:
typo, uid must match the fully qualified namespace exactly

Reply:
Done